src/software/thunderscope/robot_communication.py

```diff
@@ -7,12 +7,16 @@
 from proto.import_all_protos import *
 from pyqtgraph.Qt import QtCore
 from software.thunderscope.proto_unix_io import ProtoUnixIO
+from colorama import Fore, Style
 from typing import Type
 import threading
 import time
 import os
 from google.protobuf.message import Message
 
+DISCONNECTED = "DISCONNECTED"
+"""A constant to represent a disconnected interface"""
```

Comment:
Any reason why you're using a specific string for disconnected status instead of `None`?

Reply:
no particular reason. Setting the default interface to this constant makes it nicer to print the interfaces onto the console without needing to check for None. 

Also: I am reusing the `NetworkConfig` proto to store the different network interfaces in this class. Since it expects the parameters to be strings, setting it to `None` seemed like it would break the (implicit) protobuf contract.

I don't have a super strong opinion on this so I'm okay with using None